schema/elements.go

```diff
@@ -62,6 +63,21 @@ const (
 	Boolean = Scalar("boolean")
 )
 
+// ElementRelationship is an enum of the different possible relationships
+// between the elements of container types.
+type ElementRelationship string
```

Comment:
Topology? Didn't we spend time arguing about this word? :-)

Reply:
Did we? Do you think "ElementTopology" is clearer to devs who haven't spent a bunch of time thinking about schemas?

Reply:
No forget it, it's fine like this.